modules/ROOT/pages/authentication-authorization/built-in-roles.adoc

```diff
@@ -45,11 +45,11 @@ In other words, the `editor` role cannot add to the schema but can only make cha
 <<access-control-built-in-roles-admin, `admin`>>::
 * Access to all databases.
 * Traverse, read, and write on the data graph.
-* Allows loading data.
+* Load data.
 * Create/drop/show indexes and constraints along with any other future schema constructs.
-* Allows executing procedures using boosted privileges.
-* Allows executing admin procedures.
-* Allows executing user-defined functions using boosted privileges.
+* Execute procedures using boosted privileges.
+* Execute admin procedures.
+* Execute user-defined functions using boosted privileges.
 * View/terminate queries.
 * Manage databases, users, roles, and privileges.
 
```

Comment:
The `load data` row is duplicated in the table below

Reply:
Thanks!